AllReadyApp/Web-App/AllReady/Startup.cs

```diff
@@ -371,6 +370,23 @@ private IContainer CreateIoCContainer(IServiceCollection services)
                 await sampleData.CreateAdminUser();
             }
         }
+
+        private static void CheckForHangfireDatabaseAndCreateIfItDoesNotExist()
+        {
+            using (var sqlConnection = new SqlConnection("Server=(localdb)\\MSSQLLocalDB;Integrated Security=true;MultipleActiveResultsets=true;"))

```

Comment:
@dpaquette I wanted to use that, but the current db connections strings in config specific the hangfire db in the connection string, I don't want that, I need the connection string to not direct me to hangfire db b/c it does not exists yet. the query work is against master